src/app/components/tag-link/TagLink.tsx

```diff
@@ -65,6 +65,7 @@ export function TagLink({
       key={`${match}_${i}`}
       onClick={() => onClick(match)}
       href={getHref(match)}
+      prefetch={false}
```

Comment:
fixes this unnecessary thing
![image](https://github.com/user-attachments/assets/c6ec6a56-92a8-429d-84c6-b0ce8ebb9346)